apcd-cms/src/client/src/components/Forms/Registrations/RegistrationForm.tsx

```diff
@@ -0,0 +1,504 @@
+            const response = await fetchUtil({
+                url,
+                method: `POST`,
+                body: values,
+            });
+            if (response.status == 'success') {
+                setIsSuccess(true);
+                setRegId(response.reg_id);
+            }
+        } catch (error: any) {
+            console.error('Error saving data:', error);
+            console.log(url);
+            if (error.response && error.response.data) {
+                setErrorMessage(
+                    error.response.data.message ||
+                    'An error occurred while saving the data. Please try again.'
+                );
+            } else {
+                setErrorMessage(
+                    'An error occurred while saving the data. Please try again.'
+                );
+            }
+        } finally {
+          setSubmitting(false);
+        }      
+    };
+
+    if (isLoading) {
+        <div className="loadingField">
+            <LoadingSpinner />
+        </div>
+    }
+    
+    if (isError) {
+        return <div>Error loading form</div>;
+    }      
+    
+    return (
+        <div className="row">
+            <div className="col">
+                <h1>Request to Submit</h1>
+                <hr />
+
+                <p style={{marginBottom: '30px'}}>
+                    This form should be completed and submitted to register as a data
+                    submitter. Please review the
+                    <a
+                        href="https://sph.uth.edu/research/centers/center-for-health-care-data/assets/tx-apcd/data-submission-guides/TXAPCD%20-%20Data%20Submission%20Guide%20(DSG).pdf"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        {' '}Data Submission Guide{' '}
+                    </a>
+                    for details about completing and submitting this form, paying
+                    special attention to the schedule of submissions including test files,
+                    historical files, and monthly files.
+                </p>
+
+                <hr />
+                <Formik
+                    validateOnMount={true}
+                    initialValues={initialValues}
+                    validationSchema={validationSchema}
+                    onSubmit={handleSubmit}
+                >
+                    {({ values, isSubmitting, setFieldValue, resetForm }) => (
+                        useEffect(() => {
+                            if (isSuccess) {
+                              resetForm();
+                            }
+                        }, [isSuccess, resetForm]),
+                        isSuccess ? (
+                            <>
+                                <div style={{ marginTop: '16px', marginBottom: '16px' }}>
+                                    Your submission was successful.
+                                    Your registration ID is: <b>{{ regId }}</b>.
```

Comment:
single curly braces only, otherwise it fails in render.
```suggestion
                                    Your registration ID is: <b>{ regId }</b>.
```